functions/test/fakes.js

```diff
@@ -42,3 +43,9 @@ exports.imageManipulation = function makeFakeImageManipulation() {
     compress() { },
   });
 }
+
+exports.twitter = function makeFakeTwitter() {
+  return sinon.stub({
+    tweetMedia(mediaSize, mediaType, mediaData) { },
```

Comment:
Because `Twitter` is a class you don't need to pass in an object to `sinon.stub` like this, you could instead do
`return sinon.createStubInstance(Twitter)`. See [this](http://sinonjs.org/releases/v2.3.8/stubs/) for details.